CONTRIBUTING.md

```diff
@@ -329,6 +329,8 @@ A review can be a conceptual review, where the reviewer leaves a comment
    request",
  * `Approach (N)ACK`, meaning `Concept ACK`, but "I do (not) agree with the
    approach of this change".
+ * `NACK commit_id`, meaning `Concept ACK`, but "I do not agree with the
```

Comment:
Maybe add:

a NACK commit_id where commit_id == HEAD~N serves to NACK all commits commit_id..HEAD. Following future commits or rebases which purport to fix the issue, NACK commit_id must be explicitly removed in light of future commits which fix the issue or if the commit_id is no longer present. A commit ID is not sufficient rationale to justify a NACK. NACKs should be placed on the current tip commit ID at the time of comment, and clarifications around specific issues with non-tip commits should be described in the rationale.

Reply:
Would agree a problem is that commit IDs change all the time when people rebase, amend, etc, I think it's better to NACK by commit title or subject. Or write out your Concept ACK **but** … it seems this particular situation is rare enough I'm not sure we need a specifically documented abbreviation case for it. But maybe that's just me.

Reply:
Ok, closing for now